sniffr/breeds/seed_db_breeds.py

```diff
@@ -0,0 +1,13 @@
+import csv
+from sniffr.breeds.models import Breed, db
+
+def seed_db_breeds():
```

Comment:
Should we move this function to the seed_db.py file?

Reply:
I second this. The goal is to have one seed file that you/docker can run to start the db completely.